Update README.md (#4)
DevExpress-Examples/asp-net-core-file-manager-connect-to-database: README.md

<!-- default badges list -->
[![](https://img.shields.io/badge/Open_in_DevExpress_Support_Center-FF7200?style=flat-square&logo=DevExpress&logoColor=white)](https://supportcenter.devexpress.com/ticket/details/T828689)
[![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
<!-- default badges end -->

> # File Manager for ASP.NET Core - How to connect control to a database

Reviewer: Since File Manager (control) is the subject of this title, I think we can remove the word "control". What do you think?

Author: ```suggestion
# File Manager for ASP.NET Core - How to connect to a database
```

> This examples demonstrates how to get data from a database and display it in the [File Manager](https://docs.devexpress.com/AspNetCore/401320/devextreme-based-controls/controls/file-manager) or implement custom processing logic. To implement server interaction with a file system, pass a class that uses [file management interfaces](https://docs.devexpress.com/AspNetCore/401686/devextreme-based-controls/concepts/file-management#file-system-provider) to the [FileSystemProvider](https://docs.devexpress.com/AspNetCore/DevExtreme.AspNet.Mvc.FileManagement.FileSystemConfiguration.FileSystemProvider) property.

Reviewer: ```suggestion
This example demonstrates how to get data from a database and display it in the [File Manager](https://docs.devexpress.com/AspNetCore/401320/devextreme-based-controls/controls/file-manager) or implement custom processing logic. To implement server interaction with a file system, pass a class that uses [file management interfaces](https://docs.devexpress.com/AspNetCore/401686/devextreme-based-controls/concepts/file-management#file-system-provider) to the [FileSystemProvider](https://docs.devexpress.com/AspNetCore/DevExtreme.AspNet.Mvc.FileManagement.FileSystemConfiguration.FileSystemProvider) property.
```

## Implementation Details

> 1. Add the [FileManager](https://docs.devexpress.com/AspNetCore/401320/devextreme-based-controls/controls/file-manager) control to your page and setup it on the client side.

Reviewer: 1. "setup it on the client side" should be "set it up on the client side".

2. For my understanding, what exactly do you mean by "set it up on the client side"?

Author: ```suggestion
1. Add the [FileManager](https://docs.devexpress.com/AspNetCore/401320/devextreme-based-controls/controls/file-manager) control to your page and configure it according to your requirements.
```

3. Specify File Manager [Permissions](https://docs.devexpress.com/AspNetCore/DevExtreme.AspNet.Mvc.Builders.FileManagerBuilder.Permissions(System.Action-DevExtreme.AspNet.Mvc.Builders.FileManagerPermissionsBuilder-)?p=netframework) according to your requirements.

4. Implement required file management interfaces.

> * If you want only to display files and folders, implement [IFileSystemItemLoade](https://docs.devexpress.com/AspNetCore/DevExtreme.AspNet.Mvc.FileManagement.IFileSystemItemLoader). 

Reviewer: ```suggestion
* If you want only to display files and folders, implement [IFileSystemItemLoader](https://docs.devexpress.com/AspNetCore/DevExtreme.AspNet.Mvc.FileManagement.IFileSystemItemLoader).
```

> * If you want to copy, move, delete items - implement [IFileSystemItemEditor](https://docs.devexpress.com/AspNetCore/DevExtreme.AspNet.Mvc.FileManagement.IFileSystemItemEditor).

Reviewer: ```suggestion
* If you want to copy, move, and delete items - implement [IFileSystemItemEditor](https://docs.devexpress.com/AspNetCore/DevExtreme.AspNet.Mvc.FileManagement.IFileSystemItemEditor).
```

You can find a full implementation example in the [DbFileProvider.cs](CS/FileManagerDB/Models/DbFileProvider.cs) file.

> 5. Create a method in your [API Controller](CS/FileManagerDB/Controllers/FileManagerApiController.cs), which will handle File Manager operations. Use your custom provider there.

Reviewer: ```suggestion
5. Create a method in your [API Controller](CS/FileManagerDB/Controllers/FileManagerApiController.cs) that will handle File Manager operations. Use your custom provider there.
```
